topic-controller/src/main/java/io/strimzi/controller/topic/Controller.java

```diff
@@ -929,5 +932,12 @@ public String toString() {
     public boolean isWorkInflight() {
         return inFlight.size() > 0;
     }
+
+    /**
+     * @return an instance of BackOff with configured topic metadata max attempts
+     */
+    private BackOff getBackOff() {
```

Comment:
`getBackOff()` makes it appear to be a property/JavaBean getter, but in fact it's returning a new & stateful object each time, so I think the name is confusing. Maybe `topciMetadataBackOff()` would be clearer?

Reply:
Yeah my first name was `createTopicMetadataBackOff` then I changed in this version much simpler but I'm fine with 'topicMetadataBackOff`.